auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -263,7 +265,7 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
 
   private readonly syncBusinesses!: () => Promise<void>
   private readonly removeBusiness!: (removeBusinessPayload: RemoveBusinessPayload) => Promise<void>
-  private readonly createNumberedBusiness!: (accountId: Number) => Promise<void>
+  private readonly createNumberedBusiness!: ({ filingType, business }) => Promise<void>
```

Comment:
Apparently (prob because this method is decorated as an Action in its module), we can only have 1 formal parameter here. So I'm now passing an object with multiple properties.

Reply:
Great minds think alike!

I don't think this code will change much between now and when it's merged, so feel free to copy over what you need now. You'll still have to rebase but it will be easy.